edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html

```diff
@@ -1,56 +1,4 @@
-<div class="js-form-feedback" aria-live="assertive" tabindex="-1">
```

Comment:
Why was this block removed? @luismorenolopera

Reply:
move to login.underscore